DoctrineMigrationsBundle.php

```diff
@@ -24,4 +25,10 @@
  */
 class DoctrineMigrationsBundle extends Bundle
 {
+    /**
+     * {@inheritDoc}
+     */
+    public function registerCommands(Application $application)
```

Comment:
This can be reverted actually. See https://github.com/doctrine/DoctrineFixturesBundle/pull/205#discussion_r146107456 and https://github.com/symfony/symfony/pull/24630#discussion_r145772853

Reply:
Indeed but it avoids useless checks. See https://github.com/doctrine/DoctrineCacheBundle/pull/120#issuecomment-335927975 and https://github.com/doctrine/DoctrineBundle/pull/715#discussion_r144030327
So what should we do ?

Reply:
IMO if it works without that method should be removed without any concern for the useless check that will probably removed when the old feature is going to be phased out. Do thoses check have any measurable performance impact anyway ?

Reply:
This bundle supports outdated versions of symfony, starting from 2.3 when defining commands as services was not possible yet. This would break compatibility with such versions so either doctrine bumps symfony requirements to to a maintained version (not relevant if this is a bugfix) it must be reverted, same for DoctrineCacheBundle and DoctrineBundle.

Reply:
Thsi bundle should stop supporting unsupported version of symfony

Reply:
OK. We can keep compatibility for now. SF4 will solve this problem anyway. Reverted !

Reply:
@mikeSimonson see https://github.com/doctrine/DoctrineMigrationsBundle/pull/212